dockwidget.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""Module containing the setup for the dockwidget."""
+
+import os
+
+from PyQt4 import QtGui, uic
+from PyQt4.QtCore import pyqtSignal
+
+FORM_CLASS, _ = uic.loadUiType(os.path.join(
+    os.path.dirname(__file__), 'dockwidget.ui'))
+
+
+class Ui_DockWidget(QtGui.QDockWidget, FORM_CLASS):
```

Comment:
``Ui_DockWidget`` is probably a QGIS default name? I think a more descriptive name would make this much better.